scripts/flow_edit_matrix.py

```
@@ -0,0 +1,197 @@
+    ap = argparse.ArgumentParser()
+    ap.add_argument("--models", required=True,
+                    help="comma-sep: turbo,xl_turbo,xl_base,xl_sft,sft,base")
+    ap.add_argument("--use-cases", required=True,
+                    help=f"comma-sep: {','.join(USE_CASES)}")
+    ap.add_argument("--output-dir", default="flow_edit_matrix_outputs")
+    ap.add_argument("--device", default="cuda:0")
+    args = ap.parse_args()
+
+    models = [m.strip() for m in args.models.split(",") if m.strip()]
+    use_cases = [u.strip() for u in args.use_cases.split(",") if u.strip()]
+    out = Path(args.output_dir); out.mkdir(parents=True, exist_ok=True)
```

Ruff (0.15.12)

[error] Multiple statements on one line (semicolon)

(E702)